### PR TITLE
Is1289/mail environs

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Webserver smtp (i.e. email) section can be configured using environment variables **in production mode**  (i.e. not in devel). The new ``.env`` file can include
the following variables

```bash
SMTP_HOST=mail.speag.com
SMTP_PORT=25
SMTP_USERNAME=None
SMTP_PASSWORD=None
SMTP_TLS_ENABLED=0
```

Unfortunately this section is still very old and needs some refactoring so I took the opportunity to add some extra tests to facilitate it.

@Surfict please note that the underline library does not seem to handler properly port 587. There is a workaround but if possible try to avoid this port for the email server.
@Surfict I added some extra logs so you can trace logging into email server

## Related issue number

Implements #1289 

## Checklist

- [x] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [x] Unit tests for the changes exist
- [x] Runs in the swarm
- [x] Documentation reflects the changes
- [x] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: .env-devel [hunk 1]
```diff
@@ -39,6 +39,9 @@ S3_SECURE=0
 
 SMTP_HOST=mail.speag.com
 SMTP_PORT=25
+SMTP_USERNAME=None
+SMTP_PASSWORD=None
+SMTP_TLS_ENABLED=0
```
Comment (sanderegg) [inline .env-devel]: does this work for both tls or ssl securities?

File: services/web/server/src/simcore_service_webserver/login/__init__.py [hunk 2]
```diff
@@ -24,11 +25,37 @@
...
 
 TIMEOUT_SECS = 5
 
 
+def _create_login_config(app: web.Application, storage: AsyncpgStorage) -> Dict:
+    """
+        Creates compatible config to update login.cfg.cfg object
+    """
+    login_cfg = app[APP_CONFIG_KEY].get(CONFIG_SECTION_NAME, {}) # optional!
+    stmp_cfg = app[APP_CONFIG_KEY][SMTP_SECTION]
```
Comment (sanderegg) [inline services/web/server/src/simcore_service_webserver/login/__init__.py]: typo: smtp

File: .env-devel (hunk 1 above)
Comment (Surfict) [inline .env-devel]: Should the env-devel file edited if we use it only into production ?

Comment (pcrespov) [reply .env-devel]: Yes. I just added those as example. But on a second thought, you are right, there is no need to add variables in .env that we are not going to use that frequently

Comment (pcrespov) [reply .env-devel]: the SMTP server lib [supports direct connection via TLS/SSL](https://aiosmtplib.readthedocs.io/en/stable/usage.html#connection-options)

**Pull request merged**